exporter/doc/addresses.go

```diff
@@ -0,0 +1,53 @@
+package doc
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/projectcypress/cdatools/models"
+)
+
+// Addresses come in groups, so it should be useful to print them as a group,
+// rather than one at a time
+type Addresses struct {
+	Addresses []models.Address
+}
+
+func NewAddresses(a []models.Address) Addresses {
+	return Addresses{Addresses: a}
```

Comment:
I think these were written to provide parity to where they were needed, but if you don't think they belong even with that in mind, we can get rid of them.